### PR TITLE
README: htmlify, update links

### PR DESCRIPTION
**Pull request by jzelinskie**
- reworked heading for README including light/dark logos, tagline, new badges
- replaced features with why section focusing on value rather than functionality
- swapped youtube videos to the language primer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,9 +186,9 @@ You can try both SpiceDB and zed entirely in your browser on the [playground] th
 
 If you don't want to start with the examples loadable from the Playground, you can follow a guide for [developing a schema] or review the the schema language [design documentation].
 
-To get a quick idea of schema development, you can watch the creators of SpiceDB writing a schema for GitHub:
+To aquire an understanding of schema development, you can watch this primer video:
```
Comment (samkim) [inline README.md:189]: Typo "acquire"

```suggestion
Watch the SpiceDB Primer video to get started with schema development.
```

File: README.md [hunk 2]
```diff
@@ -210,4 +219,4 @@ We appreciate all contributions, large and small, and would like to thank all th
 In addition, we'd like to highlight a few notable contributions:
 
 - The GitHub Authorization Team for implementing and contributing the MySQL datastore
-- The Netflix Authorization Team for [sponsoring and being a design partner for Caveats](https://authzed.com/blog/abac-on-spicedb-enabling-netflix-complex-identity-types)
+- The Netflix Authorization Team for sponsoring and being a design partner for caveats
```
Comment (samkim) [inline README.md:222]: Can we add logos or at least favicon size images next to GitHub and Netflix?

Comment (jzelinskie) [reply README.md:222]: IDK if it looks like an improvement:

Screenshot 2024-02-18 at 8 53 37 PM

**Comment by jzelinskie** [comment]
ptal @josephschorr @samkim

**Review by josephschorr — APPROVED** [review]
LGTM

**Pull request merged**